python/ray/util/debugpy.py

```diff
@@ -0,0 +1,128 @@
+import logging
+import os
+import sys
+import threading
+import importlib
+
+import ray
+from ray.util.annotations import DeveloperAPI
+
+log = logging.getLogger(__name__)
+
+POST_MORTEM_ERROR_UUID = "post_mortem_error_uuid"
+
+
+def _try_import(module):
+    try:
+        return importlib.import_module(module)
+    except ModuleNotFoundError:
+        log.error(
+            f"Module '{module}' cannot be loaded. "
+            f"Ray Debugger will not work without '{module}'. "
+            f"Install this module using 'pip install {module}' "
+        )
```

Comment:
I tried throwing an exception when import error occurred, but it doesn't work well. The exception will be captured by Ray and drop the program into post-mortem mode, which will try to import debugpy again and throw another import error again... And end up with multiple levels of exception all saying there is an import error. So I choose to just log the error message and skip debugger logic if the module is not available.